tenzir/tenzir.cpp

```diff
@@ -132,10 +133,13 @@ auto main(int argc, char** argv) -> int {
     return -errno;
   }
 #endif
-  // Print the configuration file(s) that were loaded.
-  if (!cfg.config_file_path.empty()) {
-    cfg.config_files.emplace_back(std::move(cfg.config_file_path));
+  // Copy CAF detected default config file paths.
+  for (const auto& path : cfg.config_file_paths()) {
+    cfg.config_files.emplace_back(path);
   }
+  // Clear the CAF based default config file paths to avoid duplicates.
+  cfg.config_file_paths({});
```

Comment:
Should this be done in the constructor of `default_configuration` instead? Or maybe I don't understand this entirely, it's a bit hard to follow this.

Reply:
This could also be done in our overridden `configuration::parse()`, but it's also not really necessary to refactor that in a big mechanical PR like this one.